src/Microsoft.AspNetCore.Razor/TagHelpers/HtmlAttributeValueStyle.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.TagHelpers
+{
+    public enum HtmlAttributeValueStyle
+    {
+        DoubleQuotes,
+        SingleQuotes,
```

Comment:
Lots of coverage in `TagHelperBlockRewriterTest`, GitHub is just face palming on the diff size :cry: